### PR TITLE
[CHE-188] Update Docker Remove All Script

### PR DESCRIPTION
**Pull request by brok3turtl3**
### Description

This PR updates the docker-remove-all script to properly remove all images and volumes. It has also been updated to reflect recent changes in image naming conventions and react dynamically to future changes. The npm script has also been adjusted.

### Jira Task

[CHE-188](https://codehammers.atlassian.net/browse/CHE-188?atlOrigin=eyJpIjoiMjg2MTA0NTI4ODQ3NGFkYmI5MThiYjI4NmI2NTVjOTIiLCJwIjoiaiJ9)

### Testing Instructions

1. Spin up some containers using `npm run docker-dev` and `npm run docker-test:all`
2. execute the removal script using `npm run docker-remove-all`
3. Verify that all containers, images, and volumes related to Code Hammers have been removed (except the mongo image)

#### All Team Members

- [x] I added a descriptive title to this PR.
- [x] I filled out the **Description**, **Jira Task**, and **Testing Instructions** sections above.
- [ ] I added or updated [Jest unit tests]for any changes to components, server-side controllers, etc.
- [x] I ran `npm run docker-test` in my local environment to check that this PR passes all unit tests.
- [x] I did a quick check to make sure my code changes follow the recommended style guide.

### Additional Notes, Images, etc.

N/A

[CHE-188]: https://codehammers.atlassian.net/browse/CHE-188?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "docker-lint": "LINT_COMMAND=lint docker-compose -f docker-compose-lint.yml up --abort-on-container-exit",
     "docker-lint:fix": "LINT_COMMAND=lint:fix docker-compose -f docker-compose-lint.yml up --abort-on-container-exit",
     "docker-test:all": "docker-compose -f docker-compose-test.yml up --abort-on-container-exit",
-    "docker-remove-all": "bash ./scripts/docker-remove-all.sh ch ch-dev-dep-test cd-testv1 code-hammers",
+    "docker-remove-all": "bash ./scripts/docker-remove-all.sh ch ch-dev-dep code-hammers",
```
Comment (seantokuzo) [inline package.json:21]: [⛏️ Nitpick]
Edge case - if someone cloned the repo into a folder not named 'code-hammers' the volumes created when running any of the compose files will be prefixed by the folder name and not 'code-hammers' - so this script will not remove those volumes. To be safe we could prefix the volumes in each compose file:
eg. `code-hammers_node_modules` & `code-hammers_client_node_modules`
but for most that will result in ugly volume names like `code-hammers_code-hammers_node_modules`
could pick a shorter identifier to filter volumes by like `ch-vol` or something

**Review by seantokuzo — COMMENTED** [review]
This all looks great! Nice addition of the * for matching. Just one possible edge case to consider with volume removal but this lgtm

File: package.json (hunk 1 above)
Comment (brok3turtl3) [reply package.json:21]: Good observation but in this case, I'm going to rely on the fact that people clone the repo down as it is, and if they don't then the only side effect is that they lose the convenience of removing volumes with this script. They can pay the penance and manually remove them for their cloning transgressions.

**Review by seantokuzo — APPROVED** [review]
lgtm
